### ISSUE TITLE
Copying out of tl

### ISSUE THREAD

**Issue by kyrrigle** [Textualize/toolong#23]
I could see tl being super helpful but there's one thing missing that means I cannot use it for real.

Often the end result of searching for something in the logs is to copy the information out (share with coworker, create jira, etc)

It seems like there's no current way to do that?

Double clicking on a log line opens the detail.
Trying to click-drag select in detail does noting.
Ctrl-C does nothing.

If Ctrl-C could copy the current line that would be good enough but I would _love_ to be able to copy all lines found by selection.   And being able to click-drag in the detail to select part of the json would be nice.

**Comment by kyrrigle** [comment]
I meant "Command-C" to copy (Ctrl-C exits of course).   Maybe an function key to copy?

**Comment by mmoerdijk** [comment]
I agree that this would be very usefull 

With `gnome-terminal` you can use `CTRL+SHIFT+LMB`  to do a block select that way you can select the text from the detail view. 

It will introduce line breaks though

**Comment by adrfantini** [comment]
Agreed that the ability to select/copy would be absolutely needed

**Comment by rhuygen** [comment]
On macOS you can select text with the mouse while holding the Option key, then copy with CMD-C.

**Comment by spaceman7777** [comment]
Same problem here. I can't figure out any way to copy text from the terminal when using this... 

it looks great, but not being able to copy a specific bit of log file to the clipboard is a major dealbreaker.

note: i'm using xfce4-terminal w/ zsh on xubuntu 23.10

**Comment by Stubatiger** [comment]
Yes copying line/block of lines would be very helpful :) I am on Ubuntu too

**Comment by cornel-masson** [comment]
+1 for a copy function - I'm on Ubuntu under Windows (WSL).

**Comment by gergo** [comment]
+1. awsome tool so far but this would be a great addition

**Comment by anorm** [comment]
+1

I would like to extend the request to copying multiple lines at once. I currently have a stack trace of ~30 lines with several lines wider than my terminal can display. So, for me, copying individual lines is tedious.

Hence, I propose to allow a range of lines to be selected in pointer mode, and a key to copy to the system clipboard. (Similar to the following in vim: `Vjjj` then `"+y`)

**Comment by anorm** [comment]
I've created a pull request #62 which adds the single line copy function. Can be tested by installing from my branch:

```
pip install git+https://github.com/anorm/toolong.git@feature/yank-line
```
